tests/acceptance/install_test.sh

```diff
@@ -65,7 +65,7 @@ function test_install_downloads_the_non_stable_beta_version() {
     "$output"
   assert_file_exists "$installed_bashunit"
   assert_matches\
-    "$(printf "\(non-stable\) beta after ([0-9]+\.[0-9]+\.[0-9]+) \[2023-11-13\]")"\
+    "$(printf "\(non-stable\) beta after ([0-9]+\.[0-9]+\.[0-9]+) \[2023-11-13\] 🐍 \#([0-9aZ]*)")"\
```

Comment:
```suggestion
    "$(printf "\(non-stable\) beta after ([0-9]+\.[0-9]+\.[0-9]+) \[2023-11-13\] 🐍 \#([0-9aZ]{8})")"\
```
If I try the `{8}`, the pipeline fails, I don't know if there is a way to define it.

Reply:
That regex is the issue, not the `{8}` number. Also, a commit is produced using sha-1, which is a 40-character hexadecimal number. This works:
![Screenshot 2024-06-19 at 10 22 15](https://github.com/TypedDevs/bashunit/assets/5256287/060c69eb-eb91-4c56-bddc-62fb8ee1f39a)